.github/workflows/l10n.yml

```diff
@@ -20,7 +20,6 @@ jobs:
       - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
       - uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # v4.0.0
         with:
-          version: 8.15.8
```

Comment:
Remove the explicit version as it would not be updated by `renovate` and it [should not be necessary](https://github.com/pnpm/action-setup/tree/fe02b34f77f8bc703788d5817da081398fad5dd2/?tab=readme-ov-file#version) given the presence of `packageManager`:
https://github.com/mui/mui-x/blob/e92fefbf409773492250d23f0662c6d42cd874cc/package.json#L199